diff --git a/bin/merge-kits b/bin/merge-kits
index fa2526d..14fca38 100755
--- a/bin/merge-kits
+++ b/bin/merge-kits
@@ -53,6 +53,15 @@ async def main_thread(args):
 		debug=args.debug,
 		howdy=args.howdy
 	)
+
+	if args.remote_prod_url:
+		model.release_yaml.remotes['prod']['url'] = args.remote_prod_url
+
+	if args.remote_prod_mirrors == "none" and model.release_yaml.remotes['prod']['mirrors']:
+		del model.release_yaml.remotes['prod']['mirrors']
+	elif args.remote_prod_mirrors:
+		model.release_yaml.remotes['prod']['mirrors'] = args.remote_prod_mirrors.split(",")
+
 	controller = MetaRepoJobController(model, write=True)
 	try:
 		success = await controller.generate()
@@ -71,7 +80,9 @@ CLI_CONFIG = {
 	"create_branches": {"action": "store_true", "default": False},
 	"release": {"positional": True},
 	"debug": {"action": "store_true", "default": False},
-	"howdy": {"action": "store_true", "default": False}
+	"howdy": {"action": "store_true", "default": False},
+	"remote_prod_url": {"type": str, "default": None},
+	"remote_prod_mirrors": {"type": str, "default": None}
 }
